system/jlib/jfile.cpp

```diff
@@ -1984,6 +1989,16 @@ void CFileIO::setSize(offset_t pos)
 
 //-- Unix implementation ----------------------------------------------------
 
+static void sync_file_region(int fd, offset_t offset, offset_t nbytes, bool wait_previous=false)
+{
+#if defined(__linux__) && defined(__NR_sync_file_range)
+    unsigned flags = SYNC_FILE_RANGE_WRITE;
+    if (wait_previous)
+        flags |= SYNC_FILE_RANGE_WAIT_BEFORE;
+    (void)syscall(__NR_sync_file_range, fd, offset, nbytes, flags);
```

Comment:
Could you call the sync_file_range(....) function?

Reply:
done